### PR TITLE
CLOUDP-108859: Create the OnlineArchive commands under Atlas privateEndpoints

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes

_Jira ticket:_ CLOUDP-108859

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [e2e/E2E-TESTS.md](e2e/E2E-TESTS.md) (if a new command or e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by fmenezes — APPROVED** [review]
code LGTM

File: docs/command/mongocli-atlas-privateEndpoints-onlineArchive-aws-create.txt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Create a new Data Lake private endpoint for your project.
+
+When you run this command:
```
Comment (JuliaMongo) [inline docs/command/mongocli-atlas-privateEndpoints-onlineArchive-aws-create.txt:17]: @andreaangiolillo I thought we are supposed to add annotations to .go files only and these txt files get autogenerated by the build. Where is this change in the go files? WAs there a different PR? Or was this stuff there from before?

**Review by JuliaMongo — DISMISSED** [review]
LGTM, and assume that my question has to do with some changes made outside of this PR. The q was about txt files.

File: docs/command/mongocli-atlas-privateEndpoints-onlineArchive-aws-create.txt (hunk 1 above)
Comment (andreaangiolillo) [reply docs/command/mongocli-atlas-privateEndpoints-onlineArchive-aws-create.txt:17]: In this PR we are creating the command `onlineArchive` under `privateEnpoints` and then we are adding all the `privateEndpoint datalake` commands to `privateEnpoint onlineArchive`. So basically, we are reusing the same documentation of  `privateEndpoint datalake` for `privateEnpoint onlineArchive`

**Comment by johansteyn** [comment]
LGTM

**Review by johansteyn — APPROVED** [review]
LGTM

**Pull request merged**